tests/integration/galley/webhook/webhook_test.go

```diff
@@ -55,7 +55,7 @@ var (
 func setup(cfg *istio.Config) {
 	if cfg.IsIstiodEnabled() {
 		webhookControllerApp = "pilot" // i.e. istiod. Switch to 'galley' and change the setup options for non-istiod tests.
-		deployName = fmt.Sprintf("istio-%v", webhookControllerApp)
+		deployName = "istiod"
```

Comment:
`webhookControllerApp`  is still `pilot`?

Reply:
because of all the tooling that looks up on app=pilot I suggest we just rename the deployment and nothing else right now

Reply:
make sense